### PR TITLE
Add thread loader to increase build speed

### PR DESCRIPTION
**Pull request by dragonsahead**
Locally the FE builds went from ~160 seconds to ~80... but in the CI I still can't get results that blow my mind.

### CODE REVIEW

**Review by alxnddr — APPROVED** [review]
That's gives impressive 2m11s -> 48s on my machine in dev mode. Thank you, @paoliniluis 🙏

File: webpack.config.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ const config = (module.exports = {
       {
         test: /\.(tsx?|jsx?)$/,
         exclude: /node_modules|cljs/,
-        use: [{ loader: "babel-loader", options: BABEL_CONFIG }],
+        use: [
+          { loader: "thread-loader",  options: { poolRespawn: devMode? false : true } },
```
Comment (ariya) [inline webpack.config.js:70]: `poolRespawn: !devMode`

**Comment by paoliniluis** [comment]
So, we just had a talk with @alxnddr, and it makes way more sense to disable eslint rather than enabling this (at least for now), here are the reasons:
1) disabling eslint reduces WAY more the build time than thread loader with no CPU/RAM added in the process
2) the fact that I'm not seeing progress in the CI (at least for now), might be because our CI executors (most of them) have only 2 cpu's and 4gb of RAM. As the thread loader only uses 1 cpu (total cpu's - 1 to be exact) and only 4GB's of RAM, the executors might suffer from RAM starvation to which the CPU will have to waste cycles freeing up RAM or even worse, the executors will have to start swapping into disk, if there is any swap available (which is the worst nightmare ever as the disks in the CI are terribly slow)

to wrap up: this PR might serve well if we build locally (with ESlint enabled), or we use beefy executors

**Comment by ariya** [comment]
I tried to compare the use of thread-loader on GitHub-hosted runner (2 vCPU) and BuildJet with 4vCPU.
For some reasons, the result is unexpected

2 vCPU:

```
        Entrypoints:
          app-main (9.61 MiB)
              vendor.bundle.js?c8d0a21b237b7542420b
              app-main.bundle.js?1c7d60a102a6623915e6
              app-main.css?fa23f04d5e17893e6190
          app-public (7.49 MiB)
              vendor.bundle.js?c8d0a21b237b7542420b
              app-public.bundle.js?b30988ebca43cf5588bb
              app-public.css?90e24f719d1395ef2e68
          app-embed (7.49 MiB)
              vendor.bundle.js?c8d0a21b237b7542420b
              app-embed.bundle.js?7e3c1f5e2f56a995589f
              app-embed.css?90e24f719d1395ef2e68
        
        webpack 5.37.0 compiled with 2 warnings in 280653 ms
        Done in 321.27s.
```

4 vCPU:
```
        Entrypoints:
          app-main (9.61 MiB)
              vendor.bundle.js?385bb7d0a28bb1c71b78
              app-main.bundle.js?aff622cea587b21a2992
              app-main.css?fa23f04d5e17893e6190
          app-public (7.49 MiB)
              vendor.bundle.js?385bb7d0a28bb1c71b78
              app-public.bundle.js?4ab0fbd32e141430d134
              app-public.css?90e24f719d1395ef2e68
          app-embed (7.49 MiB)
              vendor.bundle.js?385bb7d0a28bb1c71b78
              app-embed.bundle.js?5ac71e59915de215bdaa
              app-embed.css?90e24f719d1395ef2e68
        
        webpack 5.37.0 compiled with 2 warnings in 289766 ms
        Done in 339.33s.
```

Could it be that the extra overhead of thread-loader (mentioned in the docs) cancels the benefit?
Is there any other reason?

**Pull request closed without merging**